assets/js/components/surveys/SurveyQuestionRating.stories.js

```diff
@@ -66,4 +66,5 @@ export default {
 			</div>
 		),
 	],
+	parameters: { padding: 0 },
```

Comment:
We need to use the default padding for this story.

Reply:
In addition to the default padding, we also need to fix this story and stories in the **`SurveyCompletion.stories.js`** file. We need to remove the custom decorator defined in the default export and move the wrapping `<div className="googlesitekit-survey"> to the template function.

Reply:
![adds-wrapper](https://user-images.githubusercontent.com/8496063/123993957-32e00f80-d9cd-11eb-9a33-40264f9fec5f.png)
The custom decorator is needed so the stories don't look this

Reply:
@danielgent oh... GH hid part of my comment. We need to move the wrapping div from the custom decorator into the template function that we use for stories. In this case, we will preserve the required markup and avoid using a custom decorator. Could you please try it on your end?

Reply:
AHhh @eugene-manuilov  right now that makes sense :smile: 

Sure no problem will do now

Reply:
Works because no VRT changes :+1: 

Back to you